src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -744,12 +745,13 @@ function ExpandedPostDetails({
             &middot;
           </NewText>
 
-          <NewText
+          <InlineLinkText
```

Comment:
This should not be a link but a button. 

The `label` is not necessary and adding it will actually break translation (e.g Google Translate) as it will not translate the `aria-label`.